Global.gd

```diff
@@ -69,34 +72,55 @@ func _ready():
-	# Item.new("Friendly Robot Servant", Item.ItemType.ALLY, "This robot will fight for you.", Enemy.RobotServant)
 	var available_items = [
 		Item.new(0, "Basic Attack", Item.ItemTier.LEVEL_ONE, Item.ItemType.MOVE, "This is a basic attack.", Moves.MoveList.BASIC_ATTACK),
 		Item.new(1, "Flimsy Sword", Item.ItemTier.LEVEL_ONE, Item.ItemType.BONUS, "This is an almost useless sword.", [Creature.Stats.ATTACK, 1]),
 		Item.new(2, "Cybernetic Eye", Item.ItemTier.LEVEL_ONE, Item.ItemType.STAT, "This eye knows where things are even if you don't.", [Creature.Stats.ACCURACY, 2]),
 		Item.new(3, "Firebolt", Item.ItemTier.LEVEL_ONE, Item.ItemType.MOVE, "This launches a bolt of fire at your enemy!", Moves.MoveList.FIREBOLT),
+		Item.new(4, "Friendly Robot Servant", Item.ItemTier.LEVEL_ONE,Item.ItemType.ALLY, "This robot will fight for you.", Enemies.EnemyList.ROBOT_T1),
+	]
+	var available_enemies = [
+		Enemy.new(1, "Guard Dog", Creature.CreatureSize.MEDIUM, 25, 25, Creature.Stats.new([1, 2, 2, 1, 1]), Creature.Stats.new([2, 0, 0, 0, 0]), Creature.BasePath.DOG, Creature.Behavior.REVENGE, [Moves.MoveList.BASIC_ATTACK]),
+		Enemy.new(2, "Mutated Dog", Creature.CreatureSize.MEDIUM, 75, 75, Creature.Stats.new([3, 4, 3, 1, 3]), Creature.Stats.new([10, 0, 0, 2, 5]), Creature.BasePath.DOG, Creature.Behavior.FOCUSED, [Moves.MoveList.BASIC_ATTACK]),
+		Enemy.new(1, "Spliced Tardigrade", Creature.CreatureSize.LARGE_TALL, 30, 30, Creature.Stats.new([1, 1, 4, 1, 4]), Creature.Stats.new([0, 4, 0, 1, 4]), Creature.BasePath.TARDIGRADE, Creature.Behavior.STUPID, [Moves.MoveList.BASIC_ATTACK]),
+		Enemy.new(2, "Gargantuan Tardigrade", Creature.CreatureSize.LARGE_TALL, 150, 150, Creature.Stats.new([5, 2, 1, 5, 0]), Creature.Stats.new([4, 4, 0, 1, 4]), Creature.BasePath.TARDIGRADE, Creature.Behavior.REVENGE, [Moves.MoveList.BASIC_ATTACK]),
+		Enemy.new(1, "Robot Servant", Creature.CreatureSize.LARGE_TALL, 35, 35, Creature.Stats.new(Creature.BASE_STATS), Creature.Stats.new(Creature.BASE_BONUSES), Creature.BasePath.ROBOT, Creature.Behavior.STUPID, [Moves.MoveList.BASIC_ATTACK]),
+		Enemy.new(2, "Robot Guard", Creature.CreatureSize.LARGE_TALL, 90, 90, Creature.Stats.new([3, 3, 3, 3, 3]), Creature.Stats.new([5, 5, 5, 5, 5]), Creature.BasePath.ROBOT, Creature.Behavior.FOCUSED, [Moves.MoveList.BASIC_ATTACK]),
+		Enemy.new(0, "Boss", Creature.CreatureSize.LARGE_TALL, 300, 300, Creature.Stats.new([4, 4, 4, 4, 4]), Creature.Stats.new([3, 3, 3, 3, 3]), Creature.BasePath.BOSS, Creature.Behavior.BOSS, [Moves.MoveList.BASIC_ATTACK]),
 	]
 
 	moves = Moves.new(available_moves)
 	items = Items.new(available_items)
+	enemies = Enemies.new(available_enemies)
 
 	# Build player after initializing everything
 	build_player()
 
 func build_player():
 	# TODO: Add upgrades
-#	var player_stats = Creature.Stats.new(Creature.BASE_STATS)
-	var player_stats = Creature.Stats.new([5,5,5,5,5])
+	var player_stats = Creature.Stats.new(Creature.BASE_STATS)
```

Comment:
The double variable names is a little confusing even though they hold different values for their purposes.